scripts/changelog.sh

```diff
@@ -85,6 +97,39 @@ collect_package_versions() {
 		pkg_name=$(echo $package | awk '{ print $1}')
 		pkg_version=$(echo $package | awk -F: '{print $2}')
 		versions[$pkg_name]=$pkg_version
+		if [ -n "$PRINT_CVE_FIXES" ]; then
+			# JSON format example:
+			# {
+			#   "version": "1",
+			#   "package": [
+			#     {
+			#       "name": "python3-cryptography",
+			#       "layer": "meta",
+			#       "version": "36.0.2",
+			#       "products": [
+			#         {
+			#           "product": "cryptography",
+			#           "cvesInRecord": "No"
+			#         }
+			#       ],
+			#       "issue": [
+			#         {
+			#           "id": "CVE-2023-23931",
+			#           "summary": "(omitted for readability)",
+			#           "scorev2": "0.0",
+			#           "scorev3": "6.5",
+			#           "vector": "NETWORK",
+			#           "status": "Patched",
+			#           "link": "https://nvd.nist.gov/vuln/detail/CVE-2023-23931"
+			#         }
+			#       ]
+			#     }
+			#   ]
+			# }
+			# collect all issue ids with status 'Unpatched' for this package:
+			cve_list=$(jq -r ".package[] | select (.name == \"${pkg_name}\") | .issue[] | select ( .status == \"Unpatched\" ) | { id } | join(\" \")" $cve_file)
```

Comment:
I have not tested if it actually works, but I find this more readable:

```suggestion
			cve_list=$(jq -r '.package[] | select (.name == "'${pkg_name}'") | .issue[] | select ( .status == "Unpatched" ) | { id } | join(" ")' $cve_file)
```

Reply:
Seems to work in a quick test